src/views/queries/requestCo2scope3.tsx

```diff
@@ -175,7 +175,7 @@ export default class Scope3CarbonConsumptionTemplates {
                   id="productId-input"
                   name="productId"
                   placeholder="BX20001"
-                  class="input-label"
+                  class="input-with-label"
```

Comment:
there's no css change in this PR. Was this a bug?

Reply:
yea, it was rendering input boxes super tiny 
<img width="727" alt="Screenshot 2024-09-11 at 09 57 26" src="https://github.com/user-attachments/assets/fa6117e7-883d-4485-98be-c27eea3eb0f0">

Reply:
good catch